### PR TITLE
It is profitable!

### PR DESCRIPTION
**Pull request by d3ck4**
So i put some enhancement to this awesome bot especially in the main trading function. Surprisingly it made profitable trades for me especially during the extremely volatile market. Thanks georgkl10 for the initial base code :)

**CHANGELOG
-----------------**
- use random module to generate random trade size, range between min and max amount setting. this is due to float to zero division exception with the initial code along with the new one.
- self.min_amount is set to 11 due to minimum quantity must be 10 usdt on Binance.
- remove the fix calculated fees as fees should be vary in each trades. (still in profit if use BNB with 0.075% discount no worries :)
- put some more outputs
- basically the main logic is to find the odd arbitrage number which is less than 0.99 not >= 1 for a potential arbitrage trade as well as the last 3rd trade must be profitable than the first initial trade for all the 3 triangular trades and thats it.
- oh i put some more coins/ticker too in the config.

Crypto to the moon!

/* end of contribution */

### CODE REVIEW

**Comment by georgk10** [comment]
Yea I've tested it over a short time period. From a technical perspective it seems to work again. However, I was not able to make a profitable trade during the tested period. 

Anyways, thanks for your contribution :)
